sway-core/src/language/ty/declaration/function.rs

```diff
@@ -240,6 +242,7 @@ impl TyFunctionDecl {
             },
             implementing_type: None,
             span,
+            call_path: CallPath::from(Ident::new_no_span("foo".into())),
```

Comment:
Perhaps a comment here might be helpful to help readers understand why an Ident with no span named foo is used?

Reply:
Moved it to an `Ident::dummy()` method since this is used elsewhere.